test/test_SW_Flow_Lib.cc

```diff
@@ -273,19 +267,1145 @@ TEST(SWFlowTest, LitterInterceptedWater) {
     }
 
     infiltrate_water_high(swc5, drain5, &drainout, pptleft, nlyrs, swcfc5, swcsat5,
-               impermeability5, &standingWater);
+      impermeability5, &standingWater);
 
     for (i = 0; i < MAX_LAYERS; i++) {
-        swc5[i] = (int)(swc5[i] * 10000000 + 0.5)/ 10000000; // need to round for unit test, despite using EXPECT_DOUBLE_EQ
-        swcsat5[i] = (int)(swcsat5[i] * 10000000 + 0.5)/ 10000000; // need to round for unit test, despite using EXPECT_DOUBLE_EQ
-        EXPECT_DOUBLE_EQ(swc5[i], swcsat5[i]); // test that swc is now equal to or below swcsat in all layers but the top
+      swc5[i] = (int)(swc5[i] * 10000000 + 0.5)/ 10000000; // need to round for unit test, despite using EXPECT_DOUBLE_EQ
+      swcsat5[i] = (int)(swcsat5[i] * 10000000 + 0.5)/ 10000000; // need to round for unit test, despite using EXPECT_DOUBLE_EQ
+      EXPECT_DOUBLE_EQ(swc5[i], swcsat5[i]); // test that swc is now equal to or below swcsat in all layers but the top
```

Comment:
Instead of rounding and then using EXPECT_DOUBLE_EQ what if we use EXPECT_NEAR instead. You test two numbers within a 'tolerance'. There is an example of its use in test_SW_SoilWater.cc